meetup/services/meetup_api_lookup_agent.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.http import HttpResponse
+import requests
+import os
+
+class MeetupApiLookupAgent():
+  def __init__(self, link):
```

Comment:
It's interesting that you're passing the meetup link as an argument to the constructor here. Maybe this isn't the `MeetupAPI`...it's something more specific to one particular meetup, like `MeetupDetails` or something. 

Some things to consider: how is this class likely to change in the future? What additional methods will you add? Will you ever try to bump it up a level of abstraction, to apply to something more general than a single meetup?